adversarial_workflow/library/models.py

```
@@ -0,0 +1,128 @@
+    categories: Dict[str, str]
+    fetched_at: Optional[datetime] = None
+
+    @classmethod
+    def from_dict(cls, data: Dict) -> "IndexData":
+        """Create an IndexData from a dictionary."""
+        evaluators = [EvaluatorEntry.from_dict(e) for e in data.get("evaluators", [])]
+        return cls(
+            version=data.get("version", "unknown"),
+            evaluators=evaluators,
+            categories=data.get("categories", {}),
+            fetched_at=datetime.now(),
```

Ruff (0.14.14)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)